vcf/invert.go

```diff
@@ -0,0 +1,46 @@
+package vcf
+
+import (
+	"log"
+	//DEBUG: "fmt"
+)
+
+//InvertGVcf inverts the reference and alt of a biallelic GVcf record.
+func InvertGVcf(g *GVcf) {
```

Comment:
I think this is beyond the scope of what anyone is currently working on, but to prevent some pernicious bugs in the future we should decide what to do what phase information here (minimally make a comment warning people). Unless InvertGVcf is used on every record in the file then we can end up flipping the haplotype of some variants but not others which could create incorrect haplotypes. I think a good way to handle this might be to have two functions: InvertAllGVcf which retains the phase information, and InvertGVcf which erases the phase information (changes "|" to "\"). I don't think this is necessary for this checkin but I do think you could add a comment and a TODO for this.

Reply:
Maybe the safe thing is to make everything unphased after lifting?
What would be the ideal solution?  Check to see if either the left or right neighbor was still the left or right neighbor after lifting.  If so, keep the phase info and otherwise convert it to unphased?